src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerManager.AnalyzerExecutionContext.cs

```diff
@@ -21,44 +23,44 @@ private sealed class AnalyzerExecutionContext
             /// <summary>
             /// Map from (symbol, analyzer) to count of its member symbols whose symbol declared events are not yet processed.
             /// </summary>
-            private Dictionary<ISymbol, HashSet<ISymbol>> _lazyPendingMemberSymbolsMapOpt;
+            private Dictionary<ISymbol, HashSet<ISymbol>?>? _lazyPendingMemberSymbolsMap;
```

Comment:
📝 We typically avoid renaming variables during the annotation PR to avoid unnecessary noise. The `Opt` suffix can be removed in a follow-up PR.

Reply:
I am not sure I agree - the naming had this suffix because file was not nullable enabled, which is being done as part of this PR, hence the rename logically belongs to this PR.

Reply:
Early in the application of NRT to Roslyn, we followed this. We stopped doing it as part of the NRT changes because it notably increased the review complexity. It's easier to focus on NRT and rename as two individual changes as opposed to trying to keep track of both at the same time.